components/lib/biometric-prompt/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,2 @@
+mock-maker-inline
```

Comment:
I believe think we do not need this any more now that we've updated our Robolectric dependency and we can start removing this and the `robolectric.properties` file from other components too (in a separate issue).

Reply:
Done